src/commands/help.rs

```diff
@@ -0,0 +1,90 @@
+use termion::event::{Event, Key};
+
+use crate::commands::{quit, CommandKeybind, KeyCommand};
+use crate::config::AppKeyMapping;
+use crate::context::{AppContext, QuitType};
+use crate::error::JoshutoResult;
+use crate::event::AppEvent;
+use crate::ui::widgets;
+use crate::ui::widgets::TuiHelp;
+use crate::ui::TuiBackend;
+
+pub fn help_loop(
+    context: &mut AppContext,
+    backend: &mut TuiBackend,
+    keymap_t: &AppKeyMapping,
+) -> JoshutoResult<()> {
+    context.flush_event();
+
+    let mut offset = 0;
+    let mut search_query = String::new();
+    let mut sort_by = 1;
+
+    while context.quit == QuitType::DoNot {
```

Comment:
You can change this to just `loop` and have a break condition (which you already have).
The user shouldn't be able to quit `joshuto` from here.